dts/arm/st/wl/stm32wl.dtsi

```diff
@@ -105,11 +105,11 @@
 			label = "STM32_CLK_RCC";
 		};
 
-		exti: interrupt-controller@58000800 {
+		exti: interrupt-controller@40010000 {
```

Comment:
There are 2 peripherals : EXTI and SYSCFG.

In the /drivers/interrupt_controller/intc_exti_stm32.c,  the **EXTI peripheral**   is actually  registered @ 0x58000800 when used by the `stm32_exti_enable` or `stm32_exti_disable`  functions.
-->  LL_EXTI_EnableIT_0_31() etc  refers to the EXTI peripheral.


This differs for stm32wl (and others stm32 mcus like stm32wb) from the **SYSCFG peripheral**  which is registered @  0x40010000 and  configured  by the   gpio_stm32_set_exti_source function (in the  drivers/gpio/gpio_stm32.c)
-->  the LL_SYSCFG_SetEXTISource(port, line) refers to the SYSCFG peripheral.

Reply:
Right, I mixed both in the debug session. Will fix, txs